finance/portfolio-performance/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        buchen portfolio 0.49.4
+name                portfolio-performance
+categories          finance
+platforms           darwin
+license             EPL-1
+maintainers         @janosch mailbox.org:janosch1 openmaintainer
+description         A simple tool to calculate the overall performance of an investment portfolio.
+long_description    An open source tool to calculate the overall performance of an investment portfolio \
+                    - across all accounts - using True-Time Weighted Return or Internal Rate of Return.
+homepage            https://www.portfolio-performance.info/en/
+
+depends_build       port:maven3 \
+                    bin:java:openjdk15
+use_configure       no
+
+# The tycho version used in PP leads to build failures. Using version 2.0.0 instead of 1.5.1 fixes it.
+# See also https://github.com/buchen/portfolio/issues/1930
+patchfiles          patch-increase-tycho-version.diff
+
+set maven_local_repository ${worksrcpath}/.m2/repository
+pre-build {
+    file mkdir ${maven_local_repository}
+}
+
+build.cmd           mvn3
+build.pre_args      clean verify -f portfolio-app -Ppackage-distro \
+                    -Dmaven.repo.local=${maven_local_repository} \
+                    -DskipTests
+
+destroot            {
+                      copy -- ${worksrcpath}/portfolio-product/target/products/name.abuchen.portfolio.product/macosx/cocoa/x86_64/PortfolioPerformance.app \
```

Comment:
Please correct me on this if I'm wrong on this, but the folder location makes it appear that this tool is built for x86_64. If this is the case, `supported_archs x86_64` should be added.

Reply:
Because of your comment I checked if it could be build for other archs. It doesent look like that other archs are supported from the upstream project. I will add your suggestion.